test/docker-compose.e2e-tests.yml

```diff
@@ -5,46 +5,73 @@ version: "3.8"
 services:
 
   grapl-e2e-tests:
-    image: grapl/grapl-e2e-tests-build:${TAG:-latest}
+    image: grapl/e2e-tests:${TAG:-latest}
     build:
-      context: ${PWD}/src
-      dockerfile: ./python/Dockerfile
-      target: grapl-e2e-tests-build
+      context: .
+      dockerfile: ./src/python/e2e-test-runner/e2e-test-runner.Dockerfile
     command: |
-      bash -c "
-        source venv/bin/activate &&
-        cd grapl_e2e_tests &&
-        python3 ./main.py
+      /bin/bash -c "
+          graplctl \
+              --grapl-region $AWS_REGION \
+              --grapl-deployment-name $DEPLOYMENT_NAME \
+              --grapl-version $DEPLOYMENT_NAME \
+              --aws-profile default \
+              upload analyzer \
+              --analyzer_main_py ./etc/local_grapl/suspicious_svchost/main.py && \
+          graplctl \
+              --grapl-region $AWS_REGION \
+              --grapl-deployment-name $DEPLOYMENT_NAME \
+              --grapl-version $DEPLOYMENT_NAME \
+              --aws-profile default \
+              upload analyzer \
+              --analyzer_main_py ./etc/local_grapl/unique_cmd_parent/main.py && \
+          graplctl \
+              --grapl-region $AWS_REGION \
+              --grapl-deployment-name $DEPLOYMENT_NAME \
+              --grapl-version $DEPLOYMENT_NAME \
+              --aws-profile default \
+              upload sysmon \
+              --logfile ./etc/sample_data/eventlog.xml && \
+          python -c 'import lambdex_handler; lambdex_handler.handler(None, None)'
```

Comment:
I assume this is due to the unzipping of the lambda package. However, why are we doing this if `graplctl` can invoke the e2e test lambda?

Reply:
`graplctl` cannot invoke the test lambda in localstack, unfortunately. It fails and no logs are emitted. I was able to get the `provisioner` lambda to run in localstack, but not the `e2e-test-runner`.